google-cloud-firestore/lib/google/cloud/firestore/client.rb

```diff
@@ -190,16 +201,37 @@ def doc document_path
         #     puts "#{city.document_id} has #{city[:population]} residents."
         #   end
         #
-        def get_all *docs
+        # @example Get docs using a field mask:
+        #   require "google/cloud/firestore"
+        #
+        #   firestore = Google::Cloud::Firestore.new
+        #
+        #   # Get and print city documents
+        #   cities = ["cities/NYC", "cities/SF", "cities/LA"]
+        #   firestore.get_all(cities, field_mask: [:population]).each do |city|
+        #     puts "#{city.document_id} has #{city[:population]} residents."
```

Comment:
Just to be clear, `document_id` not a field, correct? Is it worth a comment to explain this since otherwise the example seems to be a contradiction of the docs? ("If a document has a field that is not present in this mask, that field will not be returned in the response.")

Reply:
Correct, `document_id` is not a field. I think the code example is pretty clear. If you disagree can you make a suggestion?